src/Application/Characters/Commands/RespecializeCharacterCommand.cs

```diff
@@ -65,10 +65,7 @@ public async Task<Result<CharacterViewModel>> Handle(RespecializeCharacterComman
                 }
 
                 character.User.Gold -= price;
-                if (price == 0)
-                {
-                    character.Limitations!.LastFreeRespecializeAt = _dateTime.UtcNow;
-                }
+                character.Limitations!.LastRespecializeAt = _dateTime.UtcNow;
 
```

Comment:
Here i fixed the lastRespecAt updating.
The rest of the PR is just renaming LastFreeRespecializeAt => LastRespecializeAt
and doing the migration

Reply:
After fixing a bug, make sure you add a test that would fail without the fix.

Reply:
The fix made a test fail. I adapted the test to fit the new behavior here : 
![image](https://user-images.githubusercontent.com/47090987/236825514-c23eddaf-1671-4b66-88b3-0c07ce8be696.png)